conda_build/render.py

```diff
@@ -407,6 +407,8 @@ def get_upstream_pins(m: MetaData, precs, env):
     precs = [prec for prec in precs if prec.name in explicit_specs]
 
     ignore_pkgs_list = utils.ensure_list(m.get_value("build/ignore_run_exports_from"))
+    if m.python_version_independent and not m.noarch:
```

Comment:
IIUC, this will exclude `noarch: generic` right? (and other non-python noarch's that might exist in the future?)

Reply:
Yes, they are excluded. Only `python_version_independent: true` and `noarch: python` and `noarch_python: True` do this.